news/2019-05-01-bn-and-qat-restructure.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "BN and QAT Restructure"
+date: 2019-05-01 10:00:00 +0000
+---
+
+Changes to the ranking process are nigh! With recent discussions about [changes to the Quality Assurance Team and Beatmap Nominators](https://osu.ppy.sh/community/forums/topics/862002), we are proud to present to you the following changes.
+
+![](/wiki/shared/news/banners/beatmap-management.jpg)
+
++ The QAT is gone and has been replaced by the new [Nominator Administration Team](https://osu.ppy.sh/groups/7) (NAT). This will be a smaller, highly focused group whose name should be more representative of their responsibilities.
++ The Beatmap Nominators (BN) are responsible for disqualifying beatmaps and mediating vetoes from now on, in addition to their task of pushing beatmaps towards the ranked category.
++ Beatmaps that have been vetoed will be mediated by a random jury made up of members of the BN and NAT. This change should make quality assurance decisions more accessible to the general community.
++ The Global Moderation Team (GMT) will now handle beatmap moderation, and as such will also have access to the disqualify button. The GMT will only disqualify beatmaps in cases where they contain inappropriate content.
++ A new usergroup has been added for probationary Beatmap Nominators, while the current usergroup of BNs will have access to the disqualify button.
```

Comment:
```suggestion
+ A new usergroup has been added for probationary Beatmap Nominators.
```
reason for removing the second part is that you already said so a few points before. you may also want to add how this new usergroup will differ from regular BNs, since you're mentioning it